components/properties/src/maps.rs

```diff
@@ -289,16 +301,16 @@ impl<'a> CodePointMapDataBorrowed<'a, crate::GeneralCategory> {
     /// assert_eq!(ranges.next().unwrap(), 'µ' as u32..='µ' as u32);
     /// assert_eq!(ranges.next().unwrap(), 'º' as u32..='º' as u32);
     /// assert_eq!(ranges.next().unwrap(), 'À' as u32..='Ö' as u32);
-    /// assert_eq!(ranges.next().unwrap(), 'Ø' as u32..='Þ' as u32);
+    /// assert_eq!(ranges.next().unwrap(), 'Ø' as u32..='ö' as u32);
     /// ```
     pub fn iter_ranges_for_group(
         self,
         val: crate::GeneralCategoryGroup,
     ) -> impl Iterator<Item = RangeInclusive<u32>> + 'a {
         self.map
-            .iter_ranges()
-            .filter(move |r| val.contains(r.value))
-            .map(|r| r.range)
+            .iter_ranges_mapped(move |value| val.contains(value))
```

Comment:
suggestion: rename `value` to `group`.

The expression `val.contains(value)` seems confusing due to imprecision.